python/ray/_private/conda.py

```diff
@@ -52,3 +55,135 @@ def get_conda_bin_executable(executable_name):
         conda_bin_dir = os.path.dirname(os.environ["CONDA_EXE"])
         return os.path.join(conda_bin_dir, executable_name)
     return executable_name
+
+
+def _get_conda_env_name(conda_env_path):
```

Comment:
one question, is `conda_env_path` a config yaml?

Reply:
Yeah, it's the path to a config yaml file